### DISCUSSION TITLE
채팅방에서 필요한 정보들

### DISCUSSION THREAD

**Discussion by mypark42** [GunGonGamLeeLee/ft_transcendence#35] [3. Problem & Solving]
.

**Comment by mypark42** [comment]
채팅방 사이드 바에서 역할에 대한 정보가 필요한데,
아이디를 이용해서 직접 가져오는 것 vs 처음에 한번에 보내주는 것

**Reply by jpham005** [reply]
이거 지금은 한번에 보내주는 걸로 생각하고 있는데,
유저 수가 많아지면 id만 처음에 싹 받아오고, 해당 유저 개개인에 대한 정보가 필요할때 따로 요청하는 방식을 써야 할 것 같아요.
